### ISSUE TITLE
Error: Cannot find module 'app'

### ISSUE THREAD

**Issue by F21** [skevy/graphiql-app#29]
I am building the master on Ubuntu 16.04 with `npm 3.9.2` and `node 6.2.0`.

I am building using the following steps:
1. `npm install`
2. `npm run build`
3. `npm run package-all`

The resulting built packages does not run properly on Windows or Linux. I did not have a mac to test.

After running `GraphiQL.exe` `./GraphiQL`, I get this error:

```
A JavaScript error occurred in the main process
Uncaught Exception:
Error: Cannot find module 'app'
    at Function.Module._resolveFilename (module.js:438:15)
    at Function.Module._load (module.js:386:25)
    at Module.require (module.js:466:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/francis/Desktop/graphiql-app/release/linux-x64/GraphiQL-linux-x64/resources/app/main.js:1:90)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:456:32)
    at tryModuleLoad (module.js:415:12)
    at Function.Module._load (module.js:407:3)
```

**Comment by KyleAMathews** [comment]
Not really sure but is that how you run a Electron app?

**Comment by F21** [comment]
Yes, I was able to build a previous release a few months ago. Running `GraphiQL.exe` in the windows package worked.

I was able to find a workaround:
1. `npm install`.
2. `npm run build`. At this point, the build will get stuck on windows at 95%. See #15.
3. Cancel the build (ctrl-c).
4. Run `npm run start`.

The work around works on both Windows and Linux, but doesn't allow me to build any distributable binaries.
